### DISCUSSION TITLE
APF11 argos floats not decoding sea surface pressure correctly and so affecting DMQC for these floats

### DISCUSSION THREAD

**Discussion by ClareBellingham** [OneArgo/ADMT#85] [General]
BODC have found an issue with the decoded hex from APF11 argos floats for sea surface pressure values in that they are only decodinging as 0 or -1000 with no values of the expected +/- 2.4  available.  Teledyne have been notified but have not yet commented on their investigation into the issue.  BODC are looking for guidance on the DMQC for these floats as there are 40 floats affected and want to know if anyone else has APF11 argos floats and are seeing the same issue?

**Comment by evwcsiro** [comment]
Hi Clare, 

We have APEX Iridium APF11's at CSIRO and previously had APF9 Argos. I don't recall seeing this problem. Are you able to easily identify the problem profiles (i.e. differentiate between a true 0 pressure value and a problematic one) or are all the profiles returning those values, in which case it's more difficult? I also asked Roger Scott our RT programmer about this and he replied:

The Iridium and Argos floats are a bit different as the Iridium floats report the surface pressure offset in the system log whereas I suspect for Argos it's part of the initial/header/tech packet.  I haven't noticed a problem with our Iridium floats where it's been missing but the APF11 surface pressure stuff is messy and I'm still not comfortable with how our system handles them.  Different firmware versions seem to report the surface pressure at different times during the surface phase.  Some report it before telemetry and some after so it can be in different log files which makes assigning it to the appropriate cycle somewhat complicated.  I would be wondering for the BODC floats if it's being done after telemetry but reset at the start of the next surface phase so it's not making it into the telemetered data.  When we're missing the system log file and the float's surfaced we estimate it from the minimum of the surface measurements but it's only supposed to be a temporary measure until the file shows up.

Cheers, Esmee

**Comment by apswong** [comment]
@ClareBellingham For these APEX Argos floats with the Teledyne Apf11 controller, if the surface pressure (SP) data are truly lost and unrecoverable, then the way to proceed with DMQC of the CTD data is to follow the procedures for the APEX TNPD floats.  Please see Section 3.3.2 in the Argo QC Manual.  But please replace "TNPD ... " with something else that describes this situation in SCIENTIFIC_CALIB_COMMENT.

**Reply by ClareBellingham** [reply]
OK thank you Annie.  AOML are going to test the raw data through their decoder first to rule out a decoding issue and if we cannot recover the sea surface pressure from that we will delay mode QC the floats using this method

**Comment by cgourcuf** [comment]
Hi @ClareBellingham do you have any news about AOML tests?
